config/utils/string.js

```diff
@@ -28,7 +28,8 @@ export const isString = (str) => {
 };
 
 export const stripHTML = (str) => {
-  return (str || "").replace(/(<([^>]+)>)/gi, "");
+  if (typeof str !== "string") return str;
```

Comment:
What is being passed in instead of a string?

Reply:
In some cases, such as when we highlight keywords in search, react objects could be passed in.
I don't like this approach I did here. I'll try to move this logic a bit upward which may be a bit verbose but better suited